majifix311/src/test/java/com/example/majifix311/ProblemTest.java

```diff
@@ -141,6 +142,32 @@ public void verifyEmailIsCorrect() {
         assertFalse(builder.isValidEmail(invalid));
     }
 
+    @Test
+    public void problemsAreSortedByDate() {
```

Comment:
I'm not super convinced that this is a definitive test. With only two entries, any (visible) change, by any process, would lead to the same result. Maybe just add a couple more data points?

Reply:
You do have a point. I thought I'd add them later, when I added some other sort, or secondary sort, but since you called me on it...